code/game/objects/items/stacks/sheets/mineral.dm

```diff
@@ -106,6 +106,7 @@ GLOBAL_LIST_INIT(sandbag_recipes, list ( \
 	sheettype = "diamond"
 	materials = list(MAT_DIAMOND=MINERAL_MATERIAL_AMOUNT)
 	novariants = TRUE
+	grind_results = list("carbon" = 20)
```

Comment:
the diamond sheet doesn't have grind_results itself.

Reply:
Please explain

Reply:
Oh yeah I see now sorry I was dumb earlier.